app/services/host_fact_importer.rb

```diff
@@ -31,7 +31,9 @@ def import_facts(facts, source_proxy = nil)
       host.save(:validate => false)
     end
 
-    parse_facts(facts, type, source_proxy)
+    state = parse_facts(facts, type, source_proxy)
+    host.try(:trigger_hook, :host_facts_updated)
```

Comment:
Can `Host::Managed#trigger_hook` raise an exception? If so, should we wrap this in begin+rescue to keep the main flow working even if even triggering does not?